docs/api/ShallowWrapper/debug.md

```diff
@@ -40,14 +40,11 @@ Outputs to console:
 ```
 
 ```jsx
+const handleClick = () => {};
 const wrapper = shallow((
   <Book
     title="Huckleberry Finn"
-    cover={{
-      url: 'http://some.url/to/img.png',
-      width: 40,
-      height: 80,
-    }}
+    cover={handleClick}
```

Comment:
Yeah 💯 agree, we could make it anything other than an object due to the template issues on windows. Do you have any suggestions?